query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/simple/raw_mongo.rs

```diff
@@ -0,0 +1,233 @@
+use query_engine_tests::*;
+
+#[test_suite(schema(schema), only(MongoDb))]
+mod raw_mongo {
+    use indoc::indoc;
+    use query_engine_tests::{run_query, Runner};
+    use serde_json::json;
+
+    fn schema() -> String {
+        let schema = indoc! {
+            r#"model TestModel {
+              #id(id, Int, @id)
+              field       String
+            }"#
+        };
+
+        schema.to_owned()
+    }
+
+    #[connector_test]
+    async fn execute_raw(runner: Runner) -> TestResult<()> {
+        insta::assert_snapshot!(
+          run_query!(
+            runner,
+            run_command_raw(
+                json!({ "insert": "TestModel", "documents": [{ "_id": 1, "field": "A" }, { "_id": 2, "field": "B" }, { "_id": 3, "field": "C" }] })
+            )
+          ),
+          @r###"{"data":{"runCommandRaw":{"n":3,"ok":1.0}}}"###
```

Comment:
What was the reason we went with a different API name `runCommandRaw`? Was it just the different signature?

Conceptually it seems similar to [$executeRaw](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#executeraw)

Reply:
If `$executeRaw` doesn't make sense, we should ask others for feedback on the name. My feeling is that `runCommandRaw` is slightly verbose and should probably be flipped around `runRawCommand`. I know we did this for symmetry with the other xRaw commands, but maybe we can drop the Command altogether? `runRaw`?

Reply:
> What was the reason we went with a different API name runCommandRaw? Was it just the different signature? Conceptually it seems the same as [$executeRaw](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#executeraw), where it returns the number of affected rows.

Reasons are the following:
- It is _not_ the same as `$executeRaw` since `$executeRaw` only returns the number of affected rows. `$runCommandRaw` can be used to run, for instance, a `findAndModify` command for which you'd care about the result. Also, MongoDB doesn't have a native way of returning the number of affected rows and we don't want to get in the business of parsing responses to extract that info.
- It's also _not_ exactly the same as `$queryRaw` since you could use `$runCommandRaw` to "execute" commands for which you don't really care about the result.
- Therefore, since we already added custom raw APIs for `find` and `aggregate`, it felt more appropriate to also have a different name than `$executeRaw` or `$queryRaw`.

Reply:
I wanted to test this out because you could have finds with `$executeRaw` and inserts with `$queryRaw`

```ts
// $queryRaw insert
const insertQuery = await prisma.$queryRaw`insert into "User" (name) values ('Alice')`
// []

// $executeRaw insert
const insertExecute = await prisma.$executeRaw`insert into "User" (name) values ('Alice')`
// 1

// $queryRaw select
const selectQuery = await prisma.$queryRaw`select * from "User"`
// [
//   { id: 1, name: 'Alice' },
//   { id: 2, name: 'Alice' },
//   { id: 3, name: 'Alice' },
//   { id: 4, name: 'Alice' },
//   { id: 5, name: 'Alice' },
//   { id: 6, name: 'Alice' },
//   { id: 7, name: 'Alice' }
// ]

// $executeRaw select
const selectExecute = await prisma.$executeRaw`select * from "User"`
// 7
```

But indeed, looking at the tests, sometimes data is returned, sometimes it's not. Since we want to expose this result, it would break the existing concept of $executeRaw only returning number of affected rows and $queryRaw returning the data.

Okay, I'm sold on needing a separate operation.

Reply:
Stepping back, The reason I'm bringing this up is that I'm a bit worried we're going to have a hard time documenting and explaining this kind of API divergence. 

People understand $queryRaw and $executeRaw already, now they need to understand $runCommandRaw just for MongoDB. So if we can align the APIs, even if they're not perfect, I think that would be better. If it doesn't make sense, we'll go with what we have.

Reply:
Important to note here that they also have to learn a 100% different API for the actual query/command, so having a different method name might be exactly the right thing to make clear that this is in no way identical, besides that it runs an arbitrary/raw command.